packages/core/src/types/IKeyManager.ts

```diff
@@ -81,6 +81,35 @@ export interface IKeyManagerDecryptJWEArgs {
   data: string
 }
 
+/**
+ * Input arguments for {@link IKeyManager.keyManagerSign | keyManagerSign}
+ * @public
+ */
+export interface IKeyManagerSignArgs {
+  /**
+   * Key ID
+   */
+  kid: string
+
+  /**
+   * The algorithm to use for signing.
+   * This must be one of the algorithms supported by the KMS for this key type.
+   */
+  alg?: string
```

Comment:
This `alg` properly does not refer to the exhaustive IANA JOSE alg list, so we should use a different name, e.g., `keyOperation`. For example `eth_signTransaction` is something we only use internally and won't be registered in IANA. There will be also likely no JWT/JWS that uses alg= eth_signTransaction in the header.

Reply:
If we followed this approach, we could even go a little further and say, it is not called `Ed25519` or `EdDSA` but instead `sign_raw_eddsa` for example.

Reply:
I didn't want to add yet another name for the same thing, so to me it makes sense to call things by the same name as either IANA or ETH_RPC, or whatever the origin would be.

Using a different name instead of `alg` makes sense. I'll fix this

Reply:
renamed to `algorithm`